common/source/docs/common-geofencing-landing-page.rst

```diff
@@ -11,21 +11,14 @@ Fences
 ======
 [/site]
 
-[site wiki="plane"]
-Plane firmware prior to 4.1 supports failsafes generated by a polygonal GeoFence (boundaries described by latitude/longitude and altitude) to prevent the vehicle from traveling higher or further than desired, or into unwanted areas. See :ref:`plane:geofencing`
+ArduPilot supports a home based Cylindrical ("TinCan") and Polygonal and/or Cylindrical, Inclusion and/or Exclusion regions. Inclusion and Exclusion Fences are defined easily in Mission Planner under the PLAN screen and loaded to the autopilot, using the FENCES item in the drop down box, just like Missions or Rally Points are planned.
 
-Firmware versions 4.1 and later, support a home based Cylindrical ("TinCan") and Polygonal and/or Cylindrical, Inclusion and/or Exclusion regions. Inclusion and Exclusion Fences are defined easily in Mission Planner under the PLAN screen and loaded to the autopilot, using the FENCES item in the drop down box, just like Missions or Rally Points are planned. Upon Fence breach, selectable actions are taken.
-[/site]
-[site wiki="copter"]
-ArduPilot supports failsafes generated by several types of Fences (boundaries described by latitude/longitude and/or altitude) to prevent the vehicle from traveling higher or further than desired, or into unwanted areas. Types of Fences supported varies by vehicle. Upon Fence breach, selectable actions are taken.
-[/site]
-
-[site wiki="copter,rover"]
-For Copter, the Cylindrical Geo-Fence is a simple "tin-can" centered around HOME. In addition, there can also be inclusion and exclusion fences drawn, either circular or polygonal, or both, at various locations.
+Some vehicles can also have maximum and/or minimum altitude limits. Some flight modes will try to prevent exceeding these limits automatically, but declare a fence breach if exceeded.
```

Comment:
I might change ",but declare a fence breach if exceeded" to ". a fence breach will be declared in all modes if exceeded" or something like that.

Reply:
will do